pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -761,8 +761,9 @@ stdenv.mkDerivation (finalAttrs: {
 
   strictDeps = true;
 
-  nativeBuildInputs = [ removeReferencesTo addDriverRunpath perl pkg-config texinfo yasm ]
-  ++ optionals withCudaLLVM [ clang ];
+  nativeBuildInputs = [ removeReferencesTo addDriverRunpath perl pkg-config yasm ]
+  ++ optionals withCudaLLVM [ clang ]
+  ++ (if versionOlder version "5" then [ texinfo6 ] else [ texinfo ]);
```

@Atemu would a comment here be helpful to provide more context?
```suggestion
  # Texinfo version 7 introduced breaking changes, which older versions of ffmpeg do not handle.
  ++ (if versionOlder version "5" then [ texinfo6 ] else [ texinfo ]);
```

---

Yes.